vnext/Microsoft.ReactNative.Cxx/ReactError.h

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+#include "JSValue.h"
+
+namespace winrt::Microsoft::ReactNative::Bridge {
+
+struct ReactError {
```

Comment:
>ReactError [](start = 7, length = 10)

Could we document the fields here? E.g. I wouldn't be sure looking at this as to what "Code" or "UserInfo" represents.